src/app/migrations/datalist/datalist.2.core.txt

```diff
@@ -5,30 +5,37 @@ import { FormGroup, FormControl, Validators } from '@angular/forms';
   selector: 'app-root',
   template: `
     <form clrForm [formGroup]="model">
-      <cds-datalist status="{{ itemHasErrors ? 'error' : 'neutral' }}" control-width="shrink">
-        <label>datalist</label>
+      <cds-datalist control-width="shrink">
+        <label>Reactive Datalist</label>
```

Comment:
What is the intention behind this change?

Reply:
Simplification and most of all working inside stackblitz. Also try to keep both side equal so the focus will be on the code flow and attributes